packages/ember-metal/tests/binding/connect_test.js

```diff
@@ -100,7 +100,9 @@ testBoth('Calling connect more than once', function(get, set) {
   performTest(binding, a, b, get, set, function () {
     binding.connect(a);
 
-    binding.connect(a);
+    throws(() => {
```

Comment:
Is this an error or an assertion? If an assertion, you will need to use expectAssertion to allow the production builds to pass.